Gemfile

```diff
@@ -91,12 +65,14 @@ group :development, :test do
   gem "factory_girl_rails"
   gem "database_cleaner"
   gem "capybara-select2"
-  gem "show_me_the_cookies"
-  gem "brakeman"
+  gem 'brakeman', require: false
   gem "pry"
   gem "pry-remote"
   gem 'pry-byebug'
+  gem 'ruby-prof'
+end
+
+group :development do
```

Comment:
One thing to keep in mind is that we currently use the "test" environment for our era-test staging servers. I think this should change to use a "staging" environment but requires buy in and changes from ops. I agree that we don't need better_errors in test environment, but something to keep in mind.